examples/csharp-attribute-based-parameters-comparision/PositiveAndNegativeSimpleAttribute.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Hedgehog;
+using Hedgehog.Xunit;
+using Gen = Hedgehog.Linq.Gen;
+using Range = Hedgehog.Linq.Range;
+
+namespace csharp_attribute_based_parameters_comparision;
```

Comment:
Could this name be more general? Might be useful for other C# examples. Or heck, could even turn it into a real test suite. I should probably add it to CICD anyway - doesn't really make sense to have broken examples.

Reply:
Have updated this, I realised this when I came to do the documentation and wanted to put add the documentation examples.